docs/gettingstarted.rst

```diff
@@ -19,14 +19,33 @@ Are you looking at installing and deploying REANA cluster locally on your laptop
 
       $ sudo dpkg -i kubectl*.deb minikube*.deb
 
-2. Start Minikube virtual machine:
+2. Install Helm locally:
+
+   .. code-block:: console
+
+      $ curl https://storage.googleapis.com/kubernetes-helm/helm-v2.12.3-linux-amd64.tar.gz \
+             -o helm-v2.12.3-linux-amd64.tar.gz
+      $ tar -zxvf helm-v2.0.0-linux-amd64.tgz
+      $ mv linux-amd64/helm /usr/local/bin/helm
+
+3. Start Minikube virtual machine and then deploy Helm inside the cluster:
 
    .. code-block:: console
 
       $ minikube start --kubernetes-version="v1.12.1" \
         --feature-gates="TTLAfterFinished=true"
+      $ helm init
+
+4. Install Traefik using Helm and remember to
+   `apply the correct RBAC access control <https://github.com/containous/traefik/blob/master/docs/user-guide/kubernetes.md#role-based-access-control-configuration-kubernetes-16-only>`_
+   so Traefik works properly.
```

Comment:
Perhaps we should explicitly list the copy/pastable instruction for the RBAC step, like:

```console

Install Traefik using Helm:

$ helm install ...

Authorise Traefic to use Kubernetes API:

$ kubectl apply -f https://raw.githubusercontent.com/containous/traefik/master/examples/k8s/traefik-rbac.yaml
```

Reply:
Perhaps even with the output lines:

```console
$ kubectl apply -f https://raw.githubusercontent.com/containous/traefik/master/examples/k8s/traefik-rbac.yaml
clusterrole.rbac.authorization.k8s.io/traefik-ingress-controller created
clusterrolebinding.rbac.authorization.k8s.io/traefik-ingress-controller created
```

Reply:
In order to avoid adding this which can become outdated I've found that it can be done by passing configuration to the Helm chart, see [here](https://github.com/reanahub/reana-cluster/pull/153/files#diff-fa1e10a0359d03b37c15961dfe7d2174R2). I have also added a note on how to access the Traefik web dashboard.